### PR TITLE
Release: Include SPL Token in release tarballs

### PR DESCRIPTION
**Pull request by t-nelson**
#### Problem

Users have to go hunting for the SPL Token CLI tool

#### Summary of Changes

Include it with release tarballs

### CODE REVIEW

File: scripts/cargo-install-all.sh [hunk 1]
```diff
@@ -110,13 +110,19 @@ done
...
 
 mkdir -p "$installDir/bin"
 for bin in "${BINS[@]}"; do
   cp -fv "target/$buildVariant/$bin" "$installDir"/bin
 done
 
+splTokenBin="$(which spl-token)"
+if [[ -n "$splTokenBin" ]]; then
+  cp -fv "$splTokenBin" "$installDir"/bin
+fi
```
Comment (t-nelson) [inline scripts/cargo-install-all.sh:124]: Is it worth failing here?

File: scripts/cargo-install-all.sh [hunk 2]
```diff
@@ -110,13 +110,19 @@ done
   set -x
   # shellcheck disable=SC2086 # Don't want to double quote $rust_version
   $cargo $maybeRustVersion build $maybeReleaseFlag "${binArgs[@]}"
+  $cargo install spl-token-cli
```
Comment (mvines) [inline scripts/cargo-install-all.sh:113]:
```
cargo install spl-token-cli --root "$installDir"
```

Comment (mvines) [reply scripts/cargo-install-all.sh:113]: well, after the `mkdir -p "$installDir/bin"`

Comment (t-nelson) [reply scripts/cargo-install-all.sh:113]: Ah, nice!

**Pull request merged**